src/controllers/v1/comments.js

```diff
@@ -15,7 +15,13 @@ module.exports = class configs {
 	 */
 	async list(req) {
 		try {
-			const comments = await commentService.list(req.query.resource_id, req.decodedToken.organization_id)
+			const comments = await commentService.list(
+				req.query.resource_id,
+				req.query.page_value ? req.query.page_value : req.query.page_value,
+				req.query.context ? req.query.context : req.query.context,
+				req.decodedToken.id,
+				req.decodedToken.organization_id
```

Comment:
service file for this controlled is deleted and im not able to see a service for this. Can u check? @priyanka-TL

Reply:
added